apko_melange_build.md

```diff
@@ -0,0 +1,50 @@
+# Building KOTS with apko + melange
+
```

Comment:
Please add a section to say that this is currently used for testing the kots image and is not to be considered production ready. This ensures that someone doesn't attempt to use the image built in the pipeline within their production environments.